Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'http://rubygems.org'
 
+
 ruby '2.3.0'
 gem 'mime-types', '~> 2.6.1', require: 'mime/types/columnar'
 gem 'rails', '~> 4.2.5.2'
 gem 'rake'
+gem 'sqlite3'
```

Comment:
Can this be in a :deployment group or something so it's not installed in dev or other envs? :yellow_heart: